custom_components/tuya_local/devices/poolex_arctic_action.yaml

```diff
@@ -0,0 +1,267 @@
+name: Poolex Arctic Action
```

Comment:
I'm not sure why you moved the branded name here in your latest revision, this should be under products as it was before, with a generic name at top level. It is very common for pool heatpumps to be sold under an importer's brand, so essentially the same device is sold in different markets with different brands.